disco/extensions/upgrade_simulation/upgrades/upgrade_parameters.toml

```diff
@@ -0,0 +1,45 @@
+[thermal_upgrade_params]
+transformer_upper_limit = 1.25
+line_upper_limit = 1.25
+line_design_pu = 0.75
+transformer_design_pu = 0.75
+voltage_upper_limit = 1.05
+voltage_lower_limit = 0.95
+read_external_catalog = false
+external_catalog = ""
+
+[voltage_upgrade_params]
+initial_upper_limit = 1.05
+initial_lower_limit = 0.95
+final_upper_limit = 1.05
+final_lower_limit = 0.95
+nominal_voltage = 120
+
+[upgrade_simulation_params]
+enable_pydss_controller = true
+
+[upgrade_simulation_params.pydss_controller]
+Control1 = "VVar"
```

Comment:
Hmm. We need to avoid repeating these values. Let's decide what to do here after the team decides what curve we should be using.

Reply:
Updated per our discussion.